src/Transform/ONNX/ONNXConstProp.cpp

```diff
@@ -0,0 +1,212 @@
+//===----------- ONNXConstProp.cpp - ONNX High Level Rewriting ------------===//
+//
+// Copyright 2019-2020 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file implements a set of rewriters to constprop an ONNX operation into
+// composition of other ONNX operations.
+//
+// This pass is applied before any other pass so that there is no need to
+// implement shape inference for the constpropd operation. Hence, it is expected
+// that there is no knowledge about tensor shape at this point
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/IR/Matchers.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+#include "src/Dialect/ONNX/ONNXOps.hpp"
+#include "src/Pass/Passes.hpp"
+
+using namespace mlir;
+
+namespace {
+
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Comment style is off w.r.t. other MLIR function/class level comments. I believe it's "=" rather than "/".

Reply:
changed as requested